services/src/main/java/io/scalecube/services/ServiceRegistryImpl.java

```diff
@@ -115,7 +119,7 @@ private void loadMemberServices(DiscoveryType type, Member member) {
   /**
    * register a service instance at the cluster.
    */
-  public void registerService(Object serviceObject) {
+  public void registerService(Object serviceObject, Tag[] tags) {
```

Comment:
i think that tag is more clean solution as it describe clean data model - the format of it can be in strings or whatever.